Formula/libxcb.rb

```diff
@@ -15,7 +24,9 @@ class Libxcb < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "python@3.9" => :build
+  # Use system Python 3, to avoid a cyclic dependency on linux:
+  # python3 -> tcl-tk -> libx11 -> libxcb -> python3
+  depends_on SystemPython3Requirement => :build
```

Comment:
```suggestion
  # Use existing Python 3, to avoid a cyclic dependency on Linux:
  # python3 -> tcl-tk -> libx11 -> libxcb -> python3
  depends_on Python3Requirement => :build
```